docs/docs/deploying.md

```diff
@@ -24,26 +24,69 @@ If you're using [CronJobs()](/api/cronjob), make sure to run `quirrel ci` during
 
 :::
 
-:::note VERCEL_URL
+:::note VERCEL*URL
 If you're on Vercel, you can connect `QUIRREL_BASE_URL` to your `VERCEL_URL`.
-Only do this for preview environments, _not for production_!
+Only do this for preview environments, \_not for production*!
 `QUIRREL_BASE_URL` is used to determine the deployment that your jobs should be executed on.
 If you set it to `VERCEL_URL`, that means all jobs will be executed on the exact deployment that they were
 created on, excluding them from future bugfixes.
 :::
 
-## Hosted vs On-Prem
+## Hosted version
 
 For most people, the [hosted version](https://quirrel.dev) of Quirrel is the easiest, and probably also cheapest way of using Quirrel (there's a free tier if your project is just starting out, and OSS and side projects can apply for discounts).
 
-If you still want to host Quirrel yourself, you can do so using the [Docker Image](https://github.com/orgs/quirrel-dev/packages/container/package/quirrel). `REDIS_URL` should be set to a Redis connection string. Make sure your Redis instance is persistent. For production deployments, `PASSPHRASES` should be set to a `:`-separated list of passphrases used for securing the token endpoints. Additionally, the deployment should be secured using HTTPS.
+## How to deploy to your own server
 
-Here's a rough guideline on how to acquire tokens:
+If you still want to host Quirrel yourself, there are currently two ways to host our own quirrel server:
 
-1. Think of a good password and supply it via the `PASSPHRASES` env var.
-2. Start up your instance and run `curl --user ignored:put_password_here -X PUT localhost:9181/tokens/local`
-3. Use the obtained token for setting your application's `QUIRREL_TOKEN` env var.
+1. [By using the Docker Image](#using-the-docker-image).
+2. [By launching our own instance](#launching-our-own-instance).
```

Comment:
Using the Docker image is also a way of "Launching our own instance". Maybe a better term would be "Using the CLI"? (would also need to be changed in the headline below)